targets/ipq807x-generic

```diff
@@ -0,0 +1,11 @@
+-- Xiaomi
+
+device('xiaomi_ax3600', 'xiaomi_ax3600', {
```

Comment:
```suggestion
device('xiaomi_ax3600', 'xiaomi-ax3600', {
```

probably

Reply:
The other way around actually ;)
Should be `device('xiaomi-ax3600', 'xiaomi_ax3600', {`